message-index/messages/GHC-45696/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: If-Then-Else in pattern match
+summary: If-Then-Else in pattern match
+severity: error
+introduced: 9.6.1
+---
+
+When pattern matching, `(if .. then .. else)` blocks are not allowed as expressions to be pattern matched against.
```

Comment:
```suggestion
When pattern matching, `(if .. then .. else)` blocks are not allowed as patterns.
```
I worry that "expressions to be pattern matched against" could be interpreted as "scrutinee"

Reply:
Is the term "block" the right one here?

Reply:
I guess instead of block we could use statement here?

Reply:
s/statement/expression? :) Then I think is both clear and correct?

Reply:
I like "expression"